python/tests/unit/contents/test_chat_history.py

```diff
@@ -15,6 +15,11 @@
 from semantic_kernel.prompt_template.prompt_template_config import PromptTemplateConfig
 
 
+@pytest.fixture(scope="function")
+def chat_history():
```

Comment:
I remember doing this before, and you asked me to remove it. :)

Reply:
oops, my bad 😨